### ISSUE TITLE
Reflecting structured-buffer with a basic type error

### ISSUE THREAD

**Issue by nbenty** [shader-slang/slang#698]
Declaring this - `StructuredBuffer<float4> gStructBuff;` - causes Falcor to crash.

The culprit is the reflection, specifically doing this:
```c++
            const auto& pElementLayout = pSlangType->getElementTypeLayout();
```
`pElementLayout` is a basic type, but calling `getScalarType()` returns `None` and and rows/columns is 0.

**Comment by tfoleyNV** [comment]
Yeah, that sounds fishy.

**Comment by tfoleyNV** [comment]
Fixed by #731 and shipped in `v0.11.14`.
